### PR TITLE
[Docker] build dlc telemetry

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Create telemetry for tracking

### CODE REVIEW

File: serving/build.gradle [hunk 1]
```diff
@@ -96,7 +96,11 @@ task createDeb(type: Deb, dependsOn: distTar) {
 startScripts {
     defaultJvmOpts = []
     doLast {
+        String telemetry = 'if [[ -f "/opt/djl/bin/telemetry.sh" ]]; then\n' +
```
Comment (frankfliu) [inline serving/build.gradle:99]:
```suggestion
        String telemetry = 'if [ -f "/opt/djl/bin/telemetry.sh" ]; then\n' +
```

Comment (frankfliu) [reply serving/build.gradle:99]: this script is not using bash

File: serving/docker/pytorch-cu113.Dockerfile [hunk 2]
```diff
@@ -58,6 +60,7 @@ ARG accelerate_version=0.13.2
 
 COPY scripts scripts/
 RUN pip3 install transformers==${transformers_version} accelerate==${accelerate_version} bitsandbytes && \
+    echo "${djl_version}-transformers${transformers_version}" > /opt/djl/bin/telemetry && \
```
Comment (frankfliu) [inline serving/docker/pytorch-cu113.Dockerfile:63]: do you need create directory here?

File: serving/docker/scripts/telemetry.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/logging-and-monitoring.html
+TELEMETRY_FILE="/opt/djl/bin/telemetry"
+[[ ! -z "${OPT_OUT_TRACKING}" ]] && exit 0
+[[ ! -f "${TELEMETRY_FILE}" ]] && exit 0
+curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 600" -m 1 &>/dev/null || exit 0
+
+TOKEN=`curl -s -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 600" -m 1 2>/dev/null`
+INSTANCE_ID=`curl -s -H "X-aws-ec2-metadata-token: $TOKEN" -v http://169.254.169.254/latest/meta-data/instance-id 2>/dev/null`
+REGION=`curl -s -H "X-aws-ec2-metadata-token: $TOKEN" -v http://169.254.169.254/latest/dynamic/instance-identity/document 2>/dev/null | jq -r '.region'`
```
Comment (frankfliu) [inline serving/docker/scripts/telemetry.sh:11]: did we install jq?

File: serving/docker/pytorch-cu113.Dockerfile (hunk 2 above)
Comment (lanking520) [reply serving/docker/pytorch-cu113.Dockerfile:63]: Directory is created above

File: serving/docker/scripts/telemetry.sh (hunk 3 above)
Comment (lanking520) [reply serving/docker/scripts/telemetry.sh:11]: just installed in djl-serving

**Review by frankfliu — APPROVED** [review]

**Pull request merged**